### DISCUSSION TITLE
Unnecessary use of the ISO code for the state in labels for exemplary companies

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1399] [Business Entities (BE)]
For example to include US-DE in the name (the visible rdfs:label, used for the title in the Viewer page) of https://spec.edmcouncil.org/fibo/ontology/BE/LegalEntities/NorthAmericanEntities/USExampleEntities/TheCoca-ColaCompany-US-DE seems unnatural, not inline with business usage, and not what we'd expect real deployments of FIBO to do.
I don't think the unique labels rule need apply to exemplary Individuals like this, but even if it did, "US-DE" would not be needed to achieve uniqueness.
And this may a case where we don't have the label matching the URI.

Hence we should document some rules/conventions for example individuals generally.

**Comment by ElisaKendall** [comment]
As things stand, the functional entity is named without the jurisdiction, and legal entity is named with the jurisdiction, to support our unique label convention.  Removing the jurisdiction from the labels for the legal entity would immediately cause duplicate names/labels. 

I just fixed them all recently so that the labels and names were in sync aside from punctuation - that was hours worth of work, in fact.  It took many hours to develop these individuals.  Renaming the functional entities to call them things like 'Apple Inc issuer entity', vs. 'Apple Inc' seems really busy to me and, in cases where the same functional entity has multiple roles, such as registration authority and regulatory agency, I would have to rename all of the functional entities to include all of the roles?  I am not keen on doing that at all.

I think this is very low priority and should be closed, no change.  There is no semantic value whatsoever to making the change, and we can document a rule that says the legal entity includes the jurisdiction and functional entity does not, which is the rule I've been applying consistently for a couple of years.

**Comment by rivettp** [comment]
I agree it's not high priority, certainly not for your time, @ElisaKendall. However that does not mean it should be closed: it could be tagged for someone else as a "good first issue").
Points in response:
- I don't think the unique names rule should apply to instances, but we need to document that
- what we include in our examples should be what we can recommend for FIBO users as a best practice
- it's only just hit me, because I haven't much been looking at the examples through a UI (which tend to use the rdfs:label) but frankly it just looks odd to see the pretty cryptic-looking state code as part of the primary label. If anything the legal entity name should be the most business friendly
- now the FIBO Viewer allows graphical views of individuals (click the link for Coca Cola at the top of this issue) the odd labels are even more in your face
- and the US-DE is not included in the labels of other related individuals such as The Coca-Cola Company corporate address.
- conversely, if other individuals represent roles, then certainly it makes sense that the role(s) should be reflected in the label. It's important when people are navigating linked instances graphically for them to know what each represents
- which has already been done for the labels in cases such as https://spec.edmcouncil.org/fibo/ontology/SEC/Equities/EquitiesExampleIndividuals/TheCoca-ColaCompanyEquityIssuer so not sure how much re-labeling is really needed
- I'd hope there are few cases where the same individual really represents multiple roles, since that would indicate issues in the modeling (either of the classes or the individuals). In the example you gave is there a common role concept to represent both RegulatoryAgency and RegistrationAuthority?
